docs/frontend-api/introduction-to-frontend-api.md

```diff
@@ -160,36 +160,53 @@ Query:
         - 'QueryDecorator'   # No project-specific queries are defined.
 ```
 
-### Resolvers
-Resolvers are normal Symfony services.
-They only have to implement the `Overblog\GraphQLBundle\Definition\Resolver\QueryInterface` to be recognized as an available resolver for the GraphQL.
-
-There are several ways how to define resolvers in definition YAML files.
-We use `Overblog\GraphQLBundle\Definition\Resolver\AliasedInterface` to keep definitions simple and easy to read.
-
-`AliasedInterface` describe one method `getAliases` that should return array of `"method name" => "name used in definition"` pairs.
+### Queries
+Queries are normal Symfony services.
+They only have to extend `Shopsys\FrontendApiBundle\Model\Resolver\AbstractQuery` to be recognized as an available query for the GraphQL.
+This abstract class provides convenience for registering query functions by introducing a function naming convention for use in Query.types.yaml definitions. 
+The suffix `Query` in the name of the function is required for calling function as the alias in query decorator (`resolve: "@=query('categoriesQuery')"`). 
```

Comment:
```suggestion
This abstract class provides automatic registering of methods ending with `Query` e.g. `categoriesQuery` as aliases, so you can then use method name as `query` (`resolve: "@=query('categoriesQuery')"`) in Query.types.yaml definitions. 
```

Reply:
i thing my description is better and more descriptive.

Reply:
hm, if I had to choose, I would probably vote for Tom's suggestion, but I can live with both versions :wink: